src/frame.js

```diff
@@ -0,0 +1,30 @@
+var Frame = function() {
+  this.firstBowl = null;
+  this.secondBowl = 0;
+  this.result;
+  this.spare = false;
+  this.strike = false;
+};
+ 
+Frame.prototype.receiveBowl = function(pinsKnockedOver) {
```

Comment:
This method is too long.  Consider ways to separate the logic for handling which bowl it is from managing the result.  You have to some extent with calculateResult, but both these methods are dependent on `if/else if/else` statements which are ugly.